components/brave_sync/brave_profile_sync_service_impl.cc

```diff
@@ -200,11 +200,22 @@ BraveProfileSyncServiceImpl::BraveProfileSyncServiceImpl(Profile* profile,
                  base::Unretained(this)));
 
   model_ = BookmarkModelFactory::GetForBrowserContext(profile);
+  // model_ can be null in some tests
 
   network_connection_tracker_->AddNetworkConnectionObserver(this);
   RecordSyncStateP3A();
 }
 
+void BraveProfileSyncServiceImpl::BookmarkModelLoaded(BookmarkModel* model,
+                                                      bool ids_reassigned) {
+  DCHECK(model);
+  DCHECK(model->loaded());
+
+  LOG(WARNING) << "[BraveSync] bookmarks model just loaded, "
```

Comment:
this should be a vlog, not a warning

Reply:
fixed at https://github.com/brave/brave-core/pull/4588/files#diff-ab344835b09fb6ac1c601af787b116bbR211 and https://github.com/brave/brave-core/pull/4588/files#diff-ab344835b09fb6ac1c601af787b116bbR487